.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
 ### What does this PR do?
 
 ### What issues does this PR fix or reference?
+#<Insert GitHub Issue>, @<Insert GUS WI>@
+
+### Functionality Before
+<insert gif and/or summary>
```

Comment:
I wanted to include summary here because we also need to be mindful of accessibility. If our gifs have a lot of steps included in them, we need to make sure that everyone can understand what's happening in the mini-demo.